wagtail/admin/tests/test_date_time.py

```diff
@@ -0,0 +1,70 @@
+from django.core.checks import Error
+from django.test import TestCase, override_settings
+
+from wagtail.admin.checks import datetime_format_check
+from wagtail.test.utils import WagtailTestUtils
+
+
+class TestDateTime(WagtailTestUtils, TestCase):
+    fixtures = ["test.json"]

```

Comment:
We will need to update the tests to set `USE_L10N=True` for all tests and add a negative test with `USE_L10N=False`.

I can do this when merging.